pkgs/development/python-modules/trectools/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, beautifulsoup4
+, pythonOlder
+, pandas
+, numpy
+, scikit-learn
+, scipy
+, lxml
+, matplotlib
+, sarge
+}:
+
+buildPythonPackage rec {
+  pname = "trectools";
+  version = "0.0.49";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-WgdyFq4oyAj6oMYAwsuNyOjE7AjDFKy6UgfOtBNg5Bo=";
+  };
+
+  patches = [
+    ./bs4.patch  # use beautifulsoup4 instead of bs4
+  ];
+
+  propagatedBuildInputs = [
+    pandas
+    numpy
+    scikit-learn
+    scipy
+    lxml
+    beautifulsoup4
+    matplotlib
+    sarge
+  ];
+
```

Comment:
Do we need any checkInputs? Also please add a pythonImportsCheck.

Reply:
Added pythonImportsCheck, but as far as I can tell, by the tests succeeding, no checkInputs are needed.